frontend/src/router/router.jsx

```diff
@@ -31,6 +32,7 @@ const RouterComponent = () => { // eslint-disable-line arrow-body-style
         <Route path="/sendData" component={SendMetadata} />
         <PrivateRoute path="/articles/new/:id" component={NewExperienceArticle} />
         <PrivateRoute path="/articles/new" component={NewExperienceArticle} />
+        <Route path="/newMetadataType" component={NewMetadataType} />
```

Comment:
This should be a PrivateRoute and use the "municipality" option to only allowed municipalities to enter the route.